referenceframe/frame_system.go

```diff
@@ -308,6 +309,73 @@ func (sfs *simpleFrameSystem) FrameSystemSubset(newRoot Frame) (FrameSystem, err
 	return newFS, nil
 }
 
+// FrameSystemToPCD takes in a framesystem and returns a map where all elements are
+// the point representation of their geometry type with respect to the world.
+func FrameSystemToPCD(system FrameSystem, inputs map[string][]Input) (map[string][]r3.Vector, error) {
+	vectorMap := make(map[string][]r3.Vector)
+	// check to make sure the frame system is not empty
+	if system.FrameNames() == nil {
```

Comment:
Copy pasting Peters comment, which I also agree with 

> It's not clear to me that we should be erroring on an empty framesystem, rather than simply returning an empty  array of points. The UI would continuously throw/have to handle errors in that case for a robot until something is added to the frame system.